src/Executable.js

```diff
@@ -651,6 +651,9 @@ export default class Executable {
                 ) {
                     // Increase the backoff for the particular node and remove it from
                     // the healthy node list
+                    Logger.debug(
+                        `Node with accountId: ${node.accountId.toString()} and proxy IP: ${node.address.toString()} is unhealthy`
+                    );
```

Comment:
This error will be visible when the user enabled the logger. I will check with @Nana-EC if this is the requirement by the issue opened.